examples/manipulating_visualizing/plot_prob_atlas.py

```diff
@@ -0,0 +1,43 @@
+"""
+Plotting 4D Atlas maps or regions onto the anatomical image
+===========================================================
+
+We show here how to plot the atlas maps or statistical maps which are
+4D using a three different display types, for example using
+1. "contours", plots can be seen only in contours
```

Comment:
I would put "ROIs contours are delineated by colored lines". "plots" have no meaning in any case.